README.md

```diff
@@ -2,73 +2,250 @@
 
 The best way to handle your search bar documentation with [MeiliSearch](https://github.com/meilisearch/MeiliSearch).
 
-<!-- START doctoc generated TOC please keep comment here to allow auto update -->
-<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
-<!-- You can install de TOC plugin on VSCode and other text editors -->
+![docs-searchbar.js example](/assets/docs-searchbar-example.png)
 
+If you use Vuepress for your documentation, you might check out our [Vuepress plugin for MeiliSearch](https://github.com/meilisearch/vuepress-plugin-meilisearch).
+
+## Table of Contents <!-- omit in toc -->
+
+- [Installation and Usage](#installation-and-usage)
+  - [Before using the library](#before-using-the-library)
+  - [Basic Usage](#basic-usage)
+  - [Customization](#customization)
 - [Development Workflow](#development-workflow)
-  - [Requirements](#requirements)
-  - [Install](#install)
-  - [Build](#build)
-  - [Serve](#serve)
-  - [Test](#test)
-  - [Release](#release)
-- [Contributing](#contributing)
 - [Credits](#credits)
 
-<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+## Installation and Usage
+
+### Before using the library
+
+To integrate a search bar to your documentation and use this library, you need to scrap your website contents and to push it into a MeiliSearch instance.
```

Comment:
```suggestion
The goal of this library is to provide a very easy integration of a searchbar to your own documentation. To make that possible, you need to scrape your website content in advance, and index it in a MailiSearch instance.
```

Reply:
Minor changes.
```suggestion
The goal of this library is to provide very easy integration of a search bar into your own documentation. To make that possible, you need to scrape your website content in advance, and index it in a MeiliSearch instance.
```

Reply:
> Good! Shouldn't it be better to have a contributing.md than putting ewverything in the readme?

I let you put an issue in our repository `integration-guides` 😜

Reply:
@eskombro and @matboivin are you sure about `to scrape`?
It means "érafler". Could you update your suggestion @matboivin? 🙂

Reply:
I think this may be handled more in an organization Level as @erlend-sh suggest here:
https://github.com/meilisearch/discuss/discussions/25
So let's move that discussion there